### PR TITLE
Mattcs terraform

### PR DESCRIPTION
**Pull request by mattsmithtaiko**
This sets us up for using Terraform in the same repository by separating our development into `frontend` and `infra`. It also creates a basic Terraform file which currently does nothing. More updates will be forthcoming.

### CODE REVIEW

File: infra/infra.tf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 terraform {
   required_providers {
     aws = {
-      source = "hashicorp/aws"
+      source  = "hashicorp/aws"
```
Comment (amiersmith) [inline infra/infra.tf:4]: nit: extra space here

**Review by amiersmith — APPROVED** [review]
Looks good, great work!

**Review by cernst122 — CHANGES_REQUESTED** [review]
Can you also please update README.md, IIUC just subbing  /frontend/run-after-changing-packages.sh for ./copy-node-modules-to-local.sh?

File: frontend/Dockerfile [hunk 2]
```diff
@@ -3,8 +3,7 @@ FROM node:lts-alpine
 WORKDIR /app
 
 # Copy package and package-lock.json.
```
[the remark this answers is not in the document]
Comment (mattsmithtaiko) [reply frontend/Dockerfile]: Yes. Updated.

**Comment by amiersmith** [comment]
>

**Review by cernst122 — APPROVED** [review]

**Pull request merged**